### DISCUSSION TITLE
Disable docblock (phpDocumentor) comment block generation

### DISCUSSION THREAD

**Discussion by chw-webops** [community/community#42872] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Copilot is installed as an extension of VS Code and enabled for the PHP language. I also have the PHP Intelephense extension installed which can generate proper code Docblocks. Copilot is interfering with this by creating malformed docblocks.

Is there a way to disable just the docblock suggestions?

**Comment by emilburganov** [comment]
No(
